gdal/port/cpl_http.h

```diff
@@ -86,8 +87,17 @@ typedef struct {
 
 } CPLHTTPResult;
 
+/*! @cond Doxygen_Suppress */
+typedef size_t (CPL_STDCALL *CPLHTTPFetchWriteFunc)(void *pBuffer, size_t nSize, size_t nMemb, void *pWriteArg);
```

Comment:
this CPL_STDCALL is highly suspicious and should be removed I think. This doesn't match the signature of a curl callback. I'd suspect it to be the source of the crash with MSVC 1800 (actually due to 32 bit mode of this configuration, more than the compiler version)

Reply:
Fixed